app/policies/app_config_policy.rb

```diff
@@ -0,0 +1,30 @@
+class AppConfigPolicy < ApplicationPolicy
+  attr_reader :user, :object
+
+  def initialize(user, institution)
+    @user = user
+    @object = object
+  end
+
+  def index?
+    @user.admin?
+  end
+
+  def edit?
+    @user.admin?
+  end
+
+  def update?
+    @user.admin?
+  end
+
+  class Scope < Scope
+    def resolve
+      if @user.admin?
+        AppConfig.find
+      else
+        AppConfig.none
```

Comment:
What does `.none` do? I don't see it defined

Reply:
It comes from **ActiveRecord** and always returns an empty relation 🙂
https://apidock.com/rails/v4.0.2/ActiveRecord/QueryMethods/none